lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -135,6 +140,90 @@ def has_instructor_tab(user, course):
         )
         assert has_instructor_tab(org_researcher, self.course)
 
+    @ddt.data(
+        ('staff', False, False, True),
+        ('staff', True, False, False),
+        ('staff', True, True, True),
+    )
+    @ddt.unpack
+    def test_discussion_tab_staff(self, access_role, is_pages_and_resources_enabled,
```

Comment:
this and the following test can be ported into a single test with different ddt.data.